ballerina/json_api.bal

```diff
@@ -53,10 +53,26 @@ public isolated function parseStream(stream<byte[], error?> s, Options options =
 
 # Converts a value of type `anydata` to `json`.
 #
-# + v - Source anydata value
-# + return - representation of `v` as value of type json
-public isolated function toJson(anydata v)
-        returns json|Error = @java:Method {'class: "io.ballerina.lib.data.jsondata.json.Native"} external;
+# + j - Source anydata value
+# + return - representation of `j` as value of type json
+public isolated function toJson(anydata j) returns json {
```

Comment:
Here we are handling only the array and map right? Don't we need to handle tables?

Reply:
yes, we needs to handle `regexp` also

Reply:
As per discussed, this table and, regex won't appear in the 80% scenario therefore we will handle those with the future improvements.